wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/episode/ObtainConfirmationScreen.kt

```diff
@@ -56,7 +57,7 @@ fun ObtainConfirmationScreen(
                 Box(
                     contentAlignment = Alignment.Center,
                     modifier = Modifier
-                        .background(color = Color(0xFF202124)) // FIXME
+                        .background(color = WearColors.FF202124) // FIXME
```

Comment:
Do you still need `FIXME` comment?

Reply:
Nope, thanks for catching that. I've removed it.